content/Contributing/Applications/_index.md

```diff
@@ -53,26 +53,26 @@ Include the relevant files in the PR, including a ReadMe.txt file with any insta
 
 ## Contributing to TrueNAS Application Documentation
 
-Community members can submit change requests or add new tutorials to the **Truenas Apps > Community Apps** tutorials section of the Documentation Hub.
+Community members can now submit content directly to the [TrueNAS Apps](https://apps.truenas.com) site repository.
 
-For more information on submitting change requests, forking repos, and submitting PRs, see [Updating Content]({{< ref "/Contributing/Documentation/ContentUpdate" >}}).
+For more information about submitting a PR, see the [TrueNAS Contributor Guide](https://www.truenas.com/docs/contributing/applications/).
 
-### Submitting New Tutorial Articles
+### Submitting New or Updated Tutorial Articles
 
-When submitting new tutorials or requesting changes to existing TrueNAS Documentation Hub application tutorials:
-  
-1. Create the PR against a forked copy of the [TrueNAS Documentation Hub public repository](https://github.com/truenas/documentation/tree/master).
-  
-2. Open the existing article file in the <file>/TrueNASApps/CommunityApps</file> directory of your local forked repository.
-
-3. Add the new tutorial to the <file>CommunityApps</file> folder in your local copy of the repository if submitting a new tutorial.
+To submit new tutorials or request changes to existing application tutorials:
 
-4. Use the text editor of your choice to make changes to the [application tutorial template]({{< ref "/Contributing/Applications/AppArticleTemplate" >}}) as the basis for your new article.
-  Article content is written in Commonmark Markdown.
+1. Fork the `truenas/apps-web` repository.
+2. Create a local copy of your fork.
+3. To edit an existing application article, open the related `.md` file in the `content/catalog/` directory of your local repository copy.
+4. To submit a new tutorial or create application content, add a new `.md` file to the `content/resources/` directory. Follow the front matter format used in existing examples for consistency.
+5. If the new tutorial is associated with a catalog application entry, update the related `content/catalog/` `.md` file to link to the new tutorial.
+6. Embed images as needed by saving them in the `static/images/` directory. Follow standard Markdown image referencing procedures in your PR to ensure consistency with other articles.
+7. (Optional) Embed YouTube videos or TrueNAS Community Forum direct links in the **Resources** section. These will become visibly accessible on the **Resources** section of the app article.
```

Comment:
Would an example be helpful for guiding a contributor how to use these shortcodes so the right video loads?
```suggestion
7. (Optional) Embed YouTube videos or TrueNAS Community Forum direct links in the **Resources** section. These will become visibly accessible on the **Resources** section of the app article.
```